pkg/admin.go

```diff
@@ -150,14 +150,27 @@ func (r *Registry) RegisterEndpoint(organizationID string, id string, url string
 	if err != nil {
 		return nil, err
 	}
+	// Find out if this should be an update. That's the case if there's a RegisterEndpointEvent for the same organization
+	// and endpoint.
```

Comment:
And Identifier MUST match?

Reply:
It's considered the same endpoint when endpoint IDs match